src/Util.cc

```diff
@@ -427,15 +427,15 @@ void addResourcePaths(const std::vector<std::string> &_paths)
   char *ignPathCStr = std::getenv(systemPaths->FilePathEnv().c_str());
   if (ignPathCStr && *ignPathCStr != '\0')
   {
-    ignPaths = common::Split(ignPathCStr, ':');
+    ignPaths = common::Split(ignPathCStr, common::SystemPaths::Delimiter());
```

Comment:
Does `ign` need to be replaced with `gz` here? If not, we should update the above comment to say `Ignition file paths...` with a `Deprecated` note